test/visualTest.ts

```diff
@@ -229,6 +232,171 @@ describe("Sunburst", () => {
                     done();
                 }, 2, DefaultWaitForRender);
         });
+
+        it("data label font size should be correct", (done: DoneFn) => {
+            dataView = defaultDataViewBuilder.getDataView(
+                [
+                    defaultDataViewBuilder.RegionsDataSet,
+                    defaultDataViewBuilder.CountriesDataSet
+                ]);
+            const fontSize: number = 22;
+            const expectedFontSize: string = "22px";
+            dataView.metadata.objects = {
+                group: { labelFontSize: fontSize }
+            };
+            visualBuilder.updateRenderTimeout(
+                dataView,
+                () => {
+                    const firstClickPoint: HTMLElement = visualBuilder.slices[visualBuilder.slices.length - 1];
+                    const secondClickPoint: HTMLElement = visualBuilder.mainElement[0];
+                    d3Click(firstClickPoint, 5, 5);
+                    setTimeout(
```

Comment:
something like this might be handy here.    
   function timeout(ms: number) {
        return new Promise(resolve => setTimeout(resolve, ms));
    }

Reply:
I use that function for the new test cases but did not want to modify the old test cases. I have modified the old test cases to use that function: [615bcac](https://github.com/microsoft/powerbi-visuals-sunburst/pull/95/commits/615bcac76a6f2f971f46984c5d39e28ebce56a7f)